### PR TITLE
Add the caption label back to the insurance confirmation view but make it optional

### PR DESCRIPTION
**Pull request by robinsalehjan**
# Why?

- We decided to remove this as we were unsure if the `credit check` was something that would happen immediately after applying for the insurance.
- Bump `Podspec` to `21.0.0`

# What?

- Add the caption label, but make it optional

# Version Change

- Breaking change in the host app because the protocol introduces a new property.

# UI Changes

- Invert before/after in this [PR](https://github.com/finn-no/finnui-ios/pull/3)

### CODE REVIEW

File: FinnUI.podspec [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'FinnUI'
-  s.version      = '20.0.0'
+  s.version      = '21.0.0'
```
Comment (osanoj) [inline FinnUI.podspec:3]: Please use the fastlane release script to make new versions after the PR is merged instead 🙏

File: Sources/Fullscreen/MotorTransactionInsuranceConfirmationView/MotorTransactionInsuranceConfirmationView.swift [hunk 2]
```diff
@@ -125,6 +142,8 @@ public class MotorTransactionInsuranceConfirmationView: ShadowScrollView {
         bodyLabel.text = viewModel.bodyText
 
         keyValueGrid.configure(with: viewModel.confirmationDetails, titleStyle: .bodyStrong, valueStyle: .body)
+
+        captionLabel.text = viewModel.caption
```
Comment (osanoj) [inline Sources/Fullscreen/MotorTransactionInsuranceConfirmationView/MotorTransactionInsuranceConfirmationView.swift:146]: Doesn't it work to just set `captionLabel.isHidden = viewModel.caption == nil` here and always include it in the stack view? 🤔

Comment (robinsalehjan) [reply Sources/Fullscreen/MotorTransactionInsuranceConfirmationView/MotorTransactionInsuranceConfirmationView.swift:146]: Much better! Totally forgot about that property

**Review by osanoj — APPROVED** [review]
👍

**Pull request merged**